src/System.Net.Sockets.Legacy/tests/FunctionalTests/DisconnectAsync.cs

```diff
@@ -15,6 +15,7 @@ public void OnCompleted(object sender, SocketAsyncEventArgs args)
         }
 
         [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
Better yet, instead of comments why not add a *Nix specific test with the expectation. (Throws NotSupportedException, etc)